common/build.gradle.kts

```diff
@@ -23,6 +23,8 @@ dependencies {
     "compileOnly"("com.googlecode.json-simple:json-simple:1.1")
 
     "compileOnly"("com.google.guava:guava:30.0-jre")
+
+    "testImplementation"("com.google.guava:guava:30.0-jre")
```

Comment:
I was getting a `NoClassDefFoundError` in common's tests until I added that. I'll take a look.

Reply:
Ah. The issue is that in common, Guava is marked as `compileOnly` because in the Fabric impl, guava is available at runtime, but on Bukkit, guava is not present because... bukkit decided to be special I guess. So on Fabric, guava should be `compileOnly`, but on Bukkit, guava should be `shadedImplementation`

Reply:
Hmmm
I don't like that at all.

I'd prefer if we could do something like this:
mark it as `shadedImplementation` in common and then in fabric, we explicitly exclude it in the shadowjar. We should avoid most, if not all, compile dependencies, because then it would fuck with the standalone builds/etc.